src/main.cpp

```diff
@@ -1194,7 +1184,8 @@ bool parse_build_flags(Array<String> args) {
 							break;
 
 						case BuildFlag_UseLLVMApi:
-							build_context.use_llvm_api = true;
+							gb_printf_err("-llvm-api flag is not required any more\n");
```

Comment:
Why error out on the flag specifically? I don't remember Odin yelling at you about unknown flags?